src/dbt/kipptaf/models/people/intermediate/int_people__leadership_crosswalk.sql

```diff
@@ -41,22 +41,27 @@ select
     sl.google_email as school_leader_google_email,
     sl.job_title as school_leader_job_title,
     sl.report_to_employee_number as school_leader_report_to_employee_number,
+    sl.sam_account_name as school_leader_tableau_username,
```

Comment:
sorry: I meant rename these to `..._sam_account_name`

also, you can keep them lowercase unless that causes matching problems